index.js

```diff
@@ -28,9 +28,10 @@ function indentResolve(str, options) {
     return str;
 }
 
-module.exports = function(plugins, options) {
+module.exports = function(plugins, options, filterType) {
```

Comment:
but, why not `options.filterType`? and if filterType is Array? `['text/css', 'text/less', 'text/postcss']` or function 
```js
options.filterType = function(type) {
   return /text\/{css|less|postcss}/.test(type);
}
```